rbi/annotations/shopify-money.rbi

```diff
@@ -0,0 +1,173 @@
+# typed: strong
+
+class Money
+  sig { returns(BigDecimal) }
+  attr_reader :value
+
+  sig { returns(T.any(Money::Currency, Money::NullCurrency)) }
+  attr_reader :currency
```

Comment:
Annoyingly, `Money::NullCurrency` is not a subclass of `Money::Currency`, so we have to use a union.